tests/comdb2_files.test/runit

```diff
@@ -1,59 +1,108 @@
 #!/usr/bin/env bash
 bash -n "$0" | exit 1
 
-dbnm=$1
-dir=$DBDIR
+source ${TESTSROOTDIR}/tools/runit_common.sh
+
+function test_comdb2_files_restricted_file() {
+	(
+		# Given
+		local dbname=$1 dbdir=$2
```

Comment:
Just curious why these are locally scoped since they're the same for all the tests.